### ISSUE TITLE
This plugin work with flutter_blue?

### ISSUE THREAD

**Issue by M3sca** [pauldemarco/foreground_service#3]
Hello, this plugin work with flutter_blue?
Can I run flutter_blue, in background?
I have see many people have request this features.
Thanks!

**Comment by PrajwalDsouza** [comment]
@M3sca Yes, this plugin can work with flutter_blue as well. The only change you need to make is to go to the library files of flutter_blue and edit the **flutter_blue/android/src/main/java/com/pauldemarco/flutter_blue/FlutterBluePlugin.java** file and replace the code in it with the code given [here](https://github.com/pauldemarco/flutter_blue/blob/40d9501d5f9194dab819fc026673ca2272860eaa/android/src/main/java/com/pauldemarco/flutter_blue/FlutterBluePlugin.java).

**Comment by Eashubh128** [comment]
> Hello, this plugin work with flutter_blue? Can I run flutter_blue, in background? I have see many people have request this features. Thanks!

Hey Did you  manage to run flutter blue with foreground  service ?
